py-polars/polars/meta/versions.py

```diff
@@ -5,6 +5,31 @@
 from polars._utils.polars_version import get_polars_version
 from polars.meta.index_type import get_index_type
 
+# See the list of dependencies in pyproject.toml.
+_OPT_DEPS = [
```

Comment:
This is a bit of a micro-optimization, but I believe having this list of strings behind a function call (like it was previously) is better for the import times.

Since this is a function that is not relevant to production use, we'd rather not create this constant with every import.

Reply:
Done.